apps/rewards/contracts/Rewards.sol

```diff
@@ -97,7 +97,7 @@ contract Rewards is AragonApp {
 
         _transferReward(reward, rewardAmount);
 
-        emit RewardClaimed(_rewardID);
+        emit RewardClaimed(_rewardID, tx.origin);
```

Comment:
I am not really sure if this should be `msg.sender` instead... I invoke @Quazia for this or try to read this to ensure `tx.origin` is what we need: https://ethereum.stackexchange.com/questions/76929/tx-origin-alternative

Reply:
@ottodevs thank you for pointing this out. I decided to use `tx.origin` because in some cases the proxy address was showing up instead of the user's. But I'd need to test this more. Also, I'm gonna read about the security implications of `tx.origin`, which I was totally unaware of: https://github.com/ethereum/solidity/issues/683

Reply:
Yes. After reading about it, `tx.origin` is insecure and [not future-proof](https://ethereum.stackexchange.com/a/200/7318). I'm going to test if `msg.sender` works as expected and let you know.

Reply:
Yeah, neither one of these will behave as we want.

Reply:
@Quazia after testing `msg.sender` it does work as expected for the claim transaction event. Or am I missing something?